### DISCUSSION TITLE
Export API not returning export_id in response.

### DISCUSSION THREAD

**Discussion by aayush-eatclub** [blakeblackshear/frigate#14937] [Ask A Question] [answered]
I am using Export API.
As per documentation , it should return success , message and export_id.
<img width="445" alt="image" src="https://github.com/user-attachments/assets/ef2963ad-2f02-4b23-aa51-28ce91e40378">

But it is returning only success and message parameters , not export ID.

<img width="769" alt="image" src="https://github.com/user-attachments/assets/eb28bc69-3683-4be5-aecb-35550422de60">

I want path to local storage where this export has been stored .
Can anyone let me know how to fix this , or an alternate way to do so.
Thank You.

**Comment by NickM-27** [comment]
What version of frigate are you running?

**Reply by aayush-eatclub** [reply]
version: "3.9"

**Reply by NickM-27** [reply]
That's not a Frigate version

**Reply by aayush-eatclub** [reply]
version: "3.9"
services:
  frigate:
    container_name: frigate
    privileged: true # this may not be necessary for all setups
    restart: unless-stopped
    image: ghcr.io/blakeblackshear/frigate:stable
    shm_size: "4096mb" # update for your cameras based on calculation above
    devices:
      - /dev/dri/renderD128
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - ./config:/config
      - /home/outletmanager/outletvideocam:/media/frigate
      - type: tmpfs # Optional: 1GB of memory, reduces SSD/SD Card wear
        target: /tmp/cache
        tmpfs:
          size: 1000000000
    ports:
      - "8971:8971"
      - "5001:5000" # Internal unauthenticated access. Expose carefully.
      - "8554:8554" # RTSP feeds
      - "8555:8555/tcp" # WebRTC over tcp
      - "8555:8555/udp" # WebRTC over udp
    environment:
      FRIGATE_RTSP_PASSWORD: "password"

**Reply by NickM-27** [reply, marked as answer]
So you're using stable which is 0.14, and the code you linked is in dev which is the current release (0.15)

**Reply by aayush-eatclub** [reply]
Cool , thanks !
